tests/tests.py

```diff
@@ -55,3 +58,24 @@ def test_002_use_c4t_with_selenium(self):
         self.assertTrue(browser.title == 'Profile of p4irin · PyPI')
         time.sleep(5)
         browser.quit()
+
+    def test_003_installation_of_a_specific_version_of_assets(self):
```

Comment:
After the installation, you should verify if the switch of the active version is working with Selenium

Reply:
I'll check the browser version used after each install.

Reply:
Refactored. Used ChromeOptions and ChromeService once. Checked browser version after an assets version install.